xacml-core/src/test/java/org/xacml4j/v30/pdp/profiles/MultipleResourcesViaXPathExpressionHandlerTest.java

```diff
@@ -231,14 +231,14 @@ public void testMultipleDecisionProfileWithSelectorInMultipleCategories()
 		Attribute selector10 = r1.getOnlyEntity(Categories.RESOURCE).getOnlyAttribute(MultipleResourcesViaXPathExpressionHandler.CONTENT_SELECTOR);
 		Attribute selector11 = r1.getOnlyEntity(Categories.SUBJECT_ACCESS).getOnlyAttribute(MultipleResourcesViaXPathExpressionHandler.CONTENT_SELECTOR);
 
-		assertEquals(XPathExp.of("//md:record/md:patient[1]", Categories.RESOURCE),  Iterables.getOnlyElement(selector10.getValues()));
-		assertEquals(XPathExp.of("//md:record/md:patient[2]/md:patientDoB[1]/@md:attrn1", Categories.SUBJECT_ACCESS),  Iterables.getOnlyElement(selector11.getValues()));
+		assertEquals(XPathExp.of("//md:record/md:patient[2]", Categories.RESOURCE),  Iterables.getOnlyElement(selector10.getValues()));
```

Comment:
Why did you do this?

Reply:
@trumpyla I had to swap the assert calls to fix the failing test after guava upgrade.
I don't understand why but the XPath was captured in this sequence after upgrade.

Reply:
So instead of fixing an actual issue you fixed the test lol

Reply:
@trumpyla Sets.cartesianProduct() populates List<Category> in lexicographical order in newer version. However in older version, this order wasn't guaranteed.
https://guava.dev/releases/24.1.1-jre/api/docs/com/google/common/collect/Sets.html#cartesianProduct-java.util.List-
https://guava.dev/releases/13.0/api/docs/com/google/common/collect/Sets.html#cartesianProduct(java.util.List)